### PR TITLE
Nimitha/embed docs overview revamp

### PR DESCRIPTION
**Pull request by nimithajalal**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/index.md [hunk 1]
```diff
@@ -1,57 +1,115 @@
...
+<DiscoverBlock slots="link, text"/>
+
+[Quickstart Guide](../guides/quickstart/)
+
+Learn how to get started with the Adobe Express Embed SDK.
 
 <DiscoverBlock slots="link, text"/>
 
-[Quickstart](./quickstart/index.md)
+[Tutorials](..//guides/tutorials/)
```
Comment (undavide) [inline src/pages/guides/index.md:101]: Double slash.
Run `yarn test:links` just to be on the safe side :-)

File: src/pages/guides/index.md [hunk 2]
```diff
@@ -1,57 +1,115 @@
...
+<DiscoverBlock slots="link, text"/>
 
-## Developer Journey
+[Community](https://developer.adobe.com/express/community)
 
-1. Create a project in [Adobe Developer Console](https://adobe.io/console). Refer to the [quickstart](./quickstart/index.md) guide for these steps.
-2. Use this documentation, [code sample](https://github.com/AdobeDocs/cc-everywhere/tree/main/v4-sample/sample.md) and [demo app](https://demo.expressembed.com/) code snippets to help you get started with development.
-3. Submit your integration for [review](./review/index.md) by Adobe.
-4. Once approved, share the integration publicly with your users!
+If you need assistance or have questions about the Adobe Express Embed SDK, visit the community section for access to community forums.
```
Comment (undavide) [inline src/pages/guides/index.md:115]: Why not a link here to the community section, too...

**Review by undavide — CHANGES_REQUESTED** [review]
A few nits aside, great work! 👍🏼👍🏼👍🏼

File: src/pages/guides/index.md (hunk 2 above)
Comment (nimithajalal) [reply src/pages/guides/index.md:115]: This is linked, it is right after the discover block.

**Pull request merged**